BUILDING.md

```diff
@@ -130,6 +130,28 @@ From the grpc repository root
 $ bazel build :all
 ```
 
+## cmake: Linux, Using Make
+Run from grpc directory after cloning the repo with --recursive or updating submodules.
+```
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make
+```
+
+If you want to build shared libraries (`.so` files), run `cmake` with `-DBUILD_SHARED_LIBS=ON`.
+
+## cmake: Linux, Using Ninja (faster build)
```

Comment:
why not, but I wasn't aware that there was demand for using ninja on linux (is that based on existing issues)?

Reply:
Nothing related to gRPC specifically, but Ninja is faster than Make, and I thought it was worth mentioning, since the usage of Ninja is slightly different from Make.

Reply:
Can we move adding the linux ninja instructions to a separate pull request? I'm happy to consider it, but it feels it doesn't belong to this PR (the goal here is to try improve the quality of the existing instructions rather than adding even more ways to build)